catt/discovery.py

```diff
@@ -40,8 +40,8 @@ def get_cast_infos() -> List[pychromecast.CastInfo]:
     """
     Discover all available devices, and collect info from them.
 
-    :returns: Various device info, packed in CastInfo namedtuple.
-    :rtype: pychromecast.CastInfo
+    :returns: List of CastInfo namedtuples.
+    :rtype: List[pychromecast.CastInfo]
```

Comment:
Since we already have this information in the signature, does it make sense to duplicate it here?

Reply:
That'd be good, they were helpful when they were the only ones, but now they're just something that can go out of date.